### PR TITLE
[ODS-6560] What's new updates

### PR DESCRIPTION
**Pull request by vimayya**

### CODE REVIEW

File: odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md [hunk 1]
```diff
@@ -19,28 +19,64 @@ Ed-Fi ODS / API Platform v7.3 and provides links to additional documentation.
 
 ### Data Model Changes
 
-Ed-Fi ODS / API v7.3 adds support for [Ed-Fi Data Standard
+Ed-Fi ODS / API v7.3 adds [Ed-Fi Data Standard
 v5.2](https://edfi.atlassian.net/wiki/spaces/EFDS5/overview) implementation, with
```
Comment (stephenfuqua) [inline odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md:23]: `/reference/data-exchange/data-standard`

File: odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md [hunk 2]
```diff
@@ -19,28 +19,64 @@ Ed-Fi ODS / API Platform v7.3 and provides links to additional documentation.
...
-no breaking changes from the previous Data Standard v5.0. Refer to [What's New
+no breaking changes from the previous Data Standard v5.x. Refer to [What's New
 in Data Standard
 v5](https://edfi.atlassian.net/wiki/spaces/EFDS5/pages/26706990/What%27s+New) for
```
Comment (stephenfuqua) [inline odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md:26]: `/reference/data-exchange/data-standard/whats-new/`

File: odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md [hunk 3]
```diff
@@ -19,28 +19,64 @@ Ed-Fi ODS / API Platform v7.3 and provides links to additional documentation.
...
+mapping of identification codes used within the ecosystem. This enhancement
+simplifies API integration for clients rostered with identification codes that
+differ from identifiers used by the API host.
+
+### Token Info and Permissions
+
+Ed-Fi ODS/API v7.3 enhances the /oauth/token_info endpoint to include
+resource-level permissions, along with namespace prefixes, associated education
+organizations, and profiles assigned to a token. Refer to [Token
+Info](./../client-developers-guide/authorization.md/#token-info) for details.
```
Comment (stephenfuqua) [inline odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md:56]: `../client-developers-guide/authorization.md#token-info`

File: odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md [hunk 4]
```diff
@@ -19,28 +19,64 @@ Ed-Fi ODS / API Platform v7.3 and provides links to additional documentation.
...
 ### MetaEd IDE v4.5
 
 Implementing extensions in Ed-Fi ODS / API v7.3 requires implementers to update
-to [MetaEd IDE
-v4.5](https://edfi.atlassian.net/wiki/display/METAED20/MetaEd+Home). Refer to
-[MetaEd 4.5 What's
-New](https://edfi.atlassian.net/wiki/spaces/METAED20/pages/23711107/What%27s+New)
-for details on latest updates and improvements.
+to [MetaEd IDE v4.5](/reference/metaed). Refer to [MetaEd 4.5 What's
+New](/reference/metaed/releases/4.4.0) for details on latest updates and
```
Comment (stephenfuqua) [inline odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md:78]: Oops, I have forgotten about creating the metaed 4.5 release notes page.

File: odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md (hunk 1 above)
Comment (vimayya) [reply odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md:23]: DS space wasn't ready when I started working on this. I will update.

File: odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md (hunk 4 above)
Comment (vimayya) [reply odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md:78]: I can address updating this with rest of the post release changes(release tags and version link updates) when 4.5 is ready

**Review by roberthunterjr — APPROVED** [review]

File: odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md (hunk 4 above)
Comment (stephenfuqua) [reply odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md:78]: MetaEd 4.5 release notes are now approved in draft, ready to be published once MetaEd is published.

**Pull request merged**